src/styles/_variables.scss

```diff
@@ -10,6 +10,8 @@ $main-background-color: #000;
 $secondary-font-color: #000;
 $secondary-background-color: #fff;
 $main-links-color: lightseagreen;
+$dark-theme-background: #121212;
```

Comment:
Maybe when we implement it in heroku we can see if we change the color to a gray or another dark color so that each section can be differentiated, for example now it seems that the footer and the gallery were one but I really liked how you did the gallery

Reply:
Thank you, I think that the dark theme looks great either. But, I agree that we can try other combinations and see which one looks the best.